core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -105,6 +108,30 @@ protected static void onRetryingComplete(ClientRequestContext ctx) {
         return retryStrategy;
     }
 
+    /**
+     * Schedules next retry.
+     */
+    protected void scheduleNextRetry(ClientRequestContext ctx, Consumer<Throwable> actionOnException,
```

Comment:
- `final`
- `Consumer<? extends Throwable>`

Reply:
Fixed Thank you!